### DISCUSSION TITLE
Questions regarding field BT-3201 - Tender Identifier

### DISCUSSION THREAD

**Discussion by Fynnyan** [OP-TED/eForms-SDK#583] [Q&A] [answered]
Hello 

I got a question abut the schema definitions of field BT-3201 - Tender Identifier of an NoticeResult / LotTender.

Documentation for the field in the Ted Developer Docs seem inconstient

In the Competition Results - https://docs.ted.europa.eu/eforms/latest/schema/competition-results.html#lotTenderSection
> Tender Identifier (BT-3201) 
> An identifier of a tender. The information in the tender section refers to this tender. 
> Cardin. 1 i.e. required 
> efac:TenderReference​/cbc:ID

In the Identifier documentation - https://docs.ted.europa.eu/eforms/latest/schema/identifiers.html
> BT-3201 - Tender Identifier 
> Defined and encoded by the Contracting Authority. Should be unique between all Tender Identifiers of the procedure 
> format: No restriction yet

Checkd Rules of SDK version 1.7
>\<entry key="rule|text|BR-BT-03201-0036">BT-3201-Tender is mandatory for a notice with subtype 29 under the condition: Tender Technical ID (OPT-321-Tender) does not exist\</entry>

I looked for the rules of notice-subtype 29 & 30, where I see that this field is not required. and the rule text is confusing in the context of the other documentation.

From the different descriptions it's not clear for what that field is used, from reading the spec and the rules, it seems the field OPT-321 Tender Technical Identifier is used for identification between the sections and BT-3201 is redundant, like one of the fields mentioned in the `Pointless BTs due to technical design` section of the Identifiers

Some advice would be helpful.

**Comment by YvesJo** [comment, marked as answer]
Hi,
Indeed, "OPT-321 Tender Technical Identifier is used for identification between the sections"; its purpose is to allow to link objects together where the XML structure does not natively do it and it should follow a predefined pattern.
BT-3201, is a Business identifier and no control may be applied on its format. 
OPT-321 and BT-3201 are for Tenders what BT-137 and BT-22 are for lots.
The label rule|text|BR-BT-03201-0036 has been updated to "BT-3201-Tender is mandatory in notice type '29' **except** under following condition: 'ID – Tender' (OPT-321-Tender) is not filled in" and has been or will be released in a later version.
KR

**Reply by Fynnyan** [reply]
Thank you for the clarification.
